core/src/lib/code-generation/generated-page-object-code-generator.ts

```diff
@@ -77,66 +80,69 @@ export class GeneratedPageObjectCodeGenerator {
         return this;
     }
 
-    private addFillForm(fillForm: boolean, inputFields: E2eElement[], codeBuilder: QueuedCodeBuilder): GeneratedPageObjectCodeGenerator {
-        if (fillForm && inputFields.length > 0) {
-            codeBuilder
+    private addFillForm(params: IGeneratePageObjectParameter): GeneratedPageObjectCodeGenerator {
+        if (params.hasFillForm) {
+
+            const fillFormElements = params.elementTree.flatTree.filter(x => params.customSnippets.exists(x, 'fillForm'));
```

Comment:
würde flatTree and irgendeiner Stelle erklären, am besten direkt bei der Funktion, ist nicht trivial :D 
Aber gehört nicht zwingend in diesen pull request